views/update/remove-sold-all-land-filter.html

```diff
@@ -0,0 +1,57 @@
+{% extends "update-layout.html" %}
```

Comment:
So this is going in under "views/update", there isn't going to be a "views/remove"?

Reply:
It's a valid question, which I did consider. If we did do this then we would probably want to place all the other code assets (e.g. controllers) under a specific 'remove' folder also. I just thought, as remove journey is so closely entwined with the update journey and there are relatively few specific screens, they'd just live alongside.

What do others think?

Reply:
We could leave it for now but keep this open as the update directory is already large, so would make remove code a lot easier to find if it had its own location.

Reply:
👍  We could leave as is and keep an eye on it. Identifying the assets is important, which is why they have the 'remove-' prefix in their name.